root/static/scripts/common/entity.js

```diff
@@ -231,8 +234,23 @@ import formatTrackLength from './utility/formatTrackLength';
 
     class Label extends CoreEntity {
         selectionMessage() {
+            const code = this.label_code;
+            let codeSection = '';
+
+            if (code) {
+                codeSection = ' ' + bracketed(exp.l(
+                    'Label code: {code}',
+                    {code: formatLabelCode(code)},
+                ));
+            }
             return ReactDOMServer.renderToStaticMarkup(
-                exp.l('You selected {label}.', {label: this.reactElement({target: '_blank'})}),
+                exp.l(
+                    'You selected {label}{code}.',
```

Comment:
This works though I wonder it's easy enough for translators to make sense of. Maybe we should have two separate strings instead, `'You selected {label}.'` (keeping the no-label-code case the same) and `'You selected {label} (Label code: {code}).'` for the new case? @reosarevok @yvanzo

Reply:
Or maybe just concatenating ` + codeSection` to the result of the `ReactDOMServer.renderToStaticMarkup` call is fine, since that's basically what we do with other bracketed things.

Reply:
Rebased and pushed with this solution

Reply:
My bad, I realized now we should do the same as I suggested in https://github.com/metabrainz/musicbrainz-server/pull/1570#discussion_r443694811 so React can escape any potential HTML chars there. But I pushed the fix so we can merge this now.